settings.php

```diff
@@ -19,6 +19,12 @@
 	$error .= "There was a problem applying your settings.<br>Debugging information:<br>PHP error (".htmlspecialchars($last_error["type"])."): ".htmlspecialchars($last_error["message"])." in ".htmlspecialchars($last_error["file"]).":".htmlspecialchars($last_error["line"]);
 }
 
+# Timezone is set in docker via ENV otherwise get it from commandline
+$timezone=getenv("TZ");
```

Comment:
You are right.
When I suggested the code, I forgot the non-docker installs.

Use:

```php
$timezone = date_default_timezone_get();
```

Reply:
If I interpret https://www.php.net/manual/en/function.date-default-timezone-get.php correclty, one needs `date_default_timezone_set()` or set a TZ via `date.timezone ini option` for `date_default_timezone_get` to return anything else then `UTC` but we do not set one of the two options, or?

Reply:
Hummm... Maybe. I can test it now.

Reply:
Forget my suggestion and use the current code.